platform/broadcom/sonic-platform-modules-dell/common/dell_i2c_utils.sh

```diff
@@ -60,3 +60,60 @@ i2c_poll_bus_exists() {
     fi
 }
 
+# Perform an i2c mux device create
+# Input is of the form:
+# "echo [mux driver] <i2c-mux-address> >  <i2c-bus/operation>" i2c-channel-first
+# where operation = "new_device"
+# i2c-channel-first is the first of the 8 channels that this mux should create
+i2c_mux_create() {
+    local MAX_MUX_CHANNEL_RETRY=5
```

Comment:
I feel 5 is too large, could you change to 3.

Reply:
Addressed this